src/components/Account/Login/Login.actions.js

```diff
@@ -48,6 +49,10 @@ export function logout() {
       console.error(err);
     }
   return async dispatch => {
+    dispatch(updateNavigationSettings({ improvePhraseActive: false }));
+    try {
+      await API.updateSettings({ navigation: { improvePhraseActive: false } });
+    } catch (e) {}
```

Comment:
@tomivm Is this what you mean by:
> Also disabled it on Logout.
in the issue?

currently, it deactivates it, but when I login again, it goes back to the old status (as it was in the backend), I think this will affect users in a negative way, as they will need to activate this again on every login